packages/zoltan2/test/sphynx/Sphynx_Research_Driver.cpp

```diff
@@ -54,27 +9,37 @@
 
 #include "Teuchos_TimeMonitor.hpp" 
 #include "Teuchos_StackedTimer.hpp"
-#include "readMatrixFromBinaryFile.hpp"
+#include "ReadMatrixFromFile.hpp"
```

Comment:
This needs to be changed back to the prior.  "readMatrixFromBinaryFile.hpp" does exist in the directory. Please remove the "ReadMatrixFromFile.hpp"